Typo in getuserData.ts

Launching the electron app raises

```
Error: Cannot find module './shared/I18n'
```

This on linux because of case sensitive import (file is called i18n)

--- a/src/getUserData.ts
+++ b/src/getUserData.ts
@@ -1,7 +1,7 @@
 import * as path from "path";
 import * as fs from "fs";
 import * as electron from "electron";
-import { LocalizedError } from "./shared/I18n";
+import { LocalizedError } from "./shared/i18n";
 
 export default function(callback: (err: LocalizedError, dataPath?: string, languageCode?: string) => void) {
   let dataPathErr: LocalizedError;

